### DISCUSSION TITLE
[categorie_corrigee] Cas 6 & 11 : information VTT et montagne

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#9] [Q&A] [answered]
Dans la méthodologie, pour le cas 6 il est indiqué:

![image](https://user-images.githubusercontent.com/46813298/103222597-c81e2080-4924-11eb-9233-d6494c97c0b3.png)

Je ne trouve pas d'information de type VTT (ou montagne) dans le questionnaire. Ou est-ce que je peux trouver cette information?

**Comment by SteffMangin** [comment, marked as answer]
on ignore aujourd'hui. Les critères n'étant pas définis.
